great_expectations/experimental/datasources/postgres_datasource.py

```diff
@@ -47,19 +47,63 @@ class ColumnSplitter:
     # values are the default values if a batch request using the splitter leaves the parameter unspecified.
     # template_params: List[str]
     # Union of List/Iterable for serialization
-    param_defaults: Dict[str, Union[List, Iterable]] = Field(default_factory=dict)
+    param_defaults: Dict[str, Union[List, Iterable]] = pydantic.Field(default_factory=dict)
 
     @property
     def param_names(self) -> List[str]:
         return list(self.param_defaults.keys())
 
 
+@pydantic_dc.dataclass(frozen=True)
+class BatchSorter:
+    metadata_key: str
+    reverse: bool = False
+
+
+BatchSortersDefinition: TypeAlias = Union[List[BatchSorter], List[str]]
```

Comment:
Do we want to put type aliases at the top of the file or close by where they are used? I think the latter but just thinking out loud.

Reply:
This is really `BatchSorter` with a string shorthand. Since it refers to `BatchSorter` it must go after it is defined. I have moved up `_batch_sorter_from_list` to here to make this definition near the first use point.